modules/objdetect/src/qrcode.cpp

```diff
@@ -8,6 +8,7 @@
 #include "precomp.hpp"
 #include "opencv2/objdetect.hpp"
 #include "opencv2/calib3d.hpp"
+#include "opencv2/imgproc/types_c.h"
```

Comment:
For which types it is used?

Reply:
@rayonnant14, I don't see answer.

Reply:
@allnes @dkurt for parameters CV_RETR_TREE and CV_CHAIN_APPROX_SIMPLE in function findContours

Reply:
Please use just `RETR_TREE` and `CHAIN_APPROX_SIMPLE`